org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/tracking/IssueTrackable.java

```diff
@@ -44,17 +41,32 @@ public Integer getLine() {
 
   @Override
   public String getMessage() {
-    return SonarMarker.getMessage(issue);
+    return issue.getMessage();
+  }
+
+  @Override
+  public Integer getTextRangeHash() {
+    return textRangeHash;
   }
 
   @Override
   public Integer getLineHash() {
-    return lineHash;
+    // TODO Auto-generated method stub
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)